pkg/config/api/v1beta1/types.go

```diff
@@ -5,12 +5,11 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 )
 
-// Config contains a set of Kubernetes resources to be applied.
-// TODO: Unify with Kubernetes Config
-//       https://github.com/GoogleCloudPlatform/kubernetes/pull/1007
+// Config contains a list of Kubernetes resources to be applied.
+// DEPRECATED: All clients should use Kubernetes api.List instead of Config.
 type Config struct {
-	kapi.TypeMeta   `json:",inline"`
-	kapi.ObjectMeta `json:"metadata,omitempty"`
+	kapi.TypeMeta `json:",inline"`
+	kapi.ListMeta `json:"metadata,omitempty"`
```

Comment:
fyi, this will remove 'metadata#name' but we were not using 'name' or 'id' for Config anyway (it was optional)